src/rollup/utils/censor-tracking-keys.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Censor an object by its keys, by comparing against an array of strings and/or regexps. In the case of strings,
+ * only exact matches are censored. For non-strings, if the object's test method returns true, the key will be censored.
+ *
+ * @param {object} data the data to censor
+ * @param {obejct} keysToReject an array of strings or regexps to censor the data by preparatory to sending
+ */
+export default function censorTrackingKeys (data, keysToReject) {
+  return Object.keys(data)
+    .filter(function (key) {
+      return !keysToReject.some(function (keyToReject) {
+        return typeof keyToReject === 'string'
```

Comment:
Just checking here, so we accept mixed key types, strings and regex in the same array? I think that's a good way to go.

Reply:
correct, it detects lazily if the item is a string, so this will tolerate a mix. Here is [the spec for that case](https://github.com/lytics/pathforajs/blob/b3578284d73dc9a47abfefdb2af91a86c2104606/test/unit/utils/censor-tracking-keys.spec.js#L43)